wire-ios/Wire-iOS Tests/AVAsset/AVURLAsset+conversionTests.swift

```diff
@@ -20,6 +20,12 @@ import XCTest
 
 final class AVURLAsset_conversionTests: XCTestCase {
 
+    override class func setUp() {
+        super.setUp()
+        var flag = DeveloperFlag.proteusViaCoreCrypto
+        flag.isOn = false
+    }
```

Comment:
What do you think about changing the storage too:

```suggestion
    override class func setUp() {
        super.setUp()
        DeveloperFlag.storage = UserDefaults(suiteName: UUID().uuidString)!
        var flag = DeveloperFlag.proteusViaCoreCrypto
        flag.isOn = false
    }
    
    override class func tearDown() {
        super.tearDown()
        DeveloperFlag.storage = UserDefaults.standard
    }
```